src/react/components/connect.tsx

```diff
@@ -27,7 +27,7 @@ export class Connect extends React.Component<any> {
     }
   }
   render() {
-    return this.props.children(this.state)
+    return this.props.children({ store: this.context.store, ...this.state })
   }
```

Comment:
The problem was here. The latest release had a drawback in which the store weren't passing the store it self as a prop.